pom.xml

```diff
@@ -571,6 +571,11 @@
                 <artifactId>guava</artifactId>
                 <version>19.0</version>
             </dependency>
+            <dependency>
+                <groupId>com.googlecode.concurrentlinkedhashmap</groupId>
```

Comment:
Do not introduce this older(2015) dependency, you could check guava's MapMaker or CacheBuilder. Or, use Caffeine.